examples/dockerfiles/Kogito_swf.dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/mdessi/kogito-swf-builder:latest AS builder
+
+# Kogito User
+USER 1001
+
+# User home from base image
+WORKDIR /home/kogito/serverless-workflow-project
+
+# Copy from build context to skeleton resources project
+COPY * ./src/main/resources/
+
+# Maven vars enhirited from the base image
+RUN ${MAVEN_HOME}/bin/mvn -U -B ${MAVEN_ARGS_APPEND} -s ${MAVEN_SETTINGS_PATH} clean install -DskipTests
+
+#=============================
+# Runtime Run
+#=============================
+FROM registry.access.redhat.com/ubi8/openjdk-11:1.11
```

Comment:
```suggestion
FROM registry.access.redhat.com/ubi8/openjdk-11-runtime:latest
```

Reply:
The content of this file is to have a version to run within 50 sec instead 10 minutes

Reply:
removed due to https://issues.redhat.com/browse/KOGITO-8510